runtime/src/accounts.rs

```diff
@@ -154,8 +154,9 @@ impl Accounts {
                         let (account, rent) =
                             AccountsDB::load(storage, ancestors, accounts_index, key)
                                 .map(|(mut account, _)| {
-                                    if message.is_writable(i) && !account.executable {
-                                        let rent_due = rent_collector.update(&key, &mut account);
+                                    if message.is_writable(i) {
```

Comment:
I removed `!account.executable` because this one was fairly easy to prove it's redundant.